### PR TITLE
adding delete endpoint for hourly absence periods

### PR DESCRIPTION
**Pull request by ahmad-tayel**

### CODE REVIEW

File: personio-personnel-data-api-oa3.yaml [hunk 1]
```diff
@@ -2284,6 +2284,65 @@ paths:
...
+                  value:
+                    success: true
+                    data:
+                      message: The absence period was deleted.
+        "404":
+          description: Not found response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorAbsenceResponse"
```
Comment (rajki) [inline personio-personnel-data-api-oa3.yaml:2324]: suggestion(blocking): The schema of ErrorAbsenceResponse and ErrorDeletingAbsenceInvalidDataResponse seem to be identical. Could you pleas reuse the error response schema?

Comment (ahmad-tayel) [reply personio-personnel-data-api-oa3.yaml:2324]: but the former is a 404 not found error and the latter is a 400 bad request error, that's why I think they're complementary and not identical, but I'm thinking about renaming ErrorAbsenceResponse to ErrorAbsenceNotFoundResponse would be useful here to be more clear what these errors represent, what do you think?

Comment (ahmad-tayel) [reply personio-personnel-data-api-oa3.yaml:2324]: Ahh sorry true I got your point, changing it 👍

Comment (ahmad-tayel) [reply personio-personnel-data-api-oa3.yaml:2324]: I just got confused as I saw the same schema duplicated for different errors in absences like `ErrorInsertingAbsenceResponse` and `ErrorCreateAbsenceResponse`, so I thought we're adopting this style 👍

Comment (rajki) [reply personio-personnel-data-api-oa3.yaml:2324]: Fair point, we should think about consolidating the error response schema. But this is out of scope of this patch 😄

**Review by rajki — APPROVED** [review]

**Pull request merged**